### ISSUE TITLE
After starting katello services, hammer ping fails as foreman_tasks doesn't comes up immediately

### ISSUE THREAD

**Issue by sghai** [RedHatSatellite/satellite-clone#46]
```console
TASK [sat6repro : Restart katello-service and test satellite] ******************
changed: [Server] => (item=katello-service start)
failed: [Server] (item=hammer ping) => {"changed": true, "cmd": ["hammer", "ping"], "delta": "0:00:54.061257", "end": "2017-01-10 04:35:41.361823", "failed": true, "item": "hammer ping", "rc": 1, "start": "2017-01-10 04:34:47.300566", "stderr": "", "stdout": "candlepin:      \n    Status:          ok\n    Server Response: Duration: 35ms\ncandlepin_auth: \n    Status:          ok\n    Server Response: Duration: 36ms\npulp:           \n    Status:          ok\n    Server Response: Duration: 63ms\nforeman_tasks:  \n    Status:          FAIL\n    Server Response:", "stdout_lines": ["candlepin:      ", "    Status:          ok", "    Server Response: Duration: 35ms", "candlepin_auth: ", "    Status:          ok", "    Server Response: Duration: 36ms", "pulp:           ", "    Status:          ok", "    Server Response: Duration: 63ms", "foreman_tasks:  ", "    Status:          FAIL", "    Server Response:"], "warnings": []}
```

This appears only when we use a large db and it might be because when multiple foreman_tasks are running in backend

**Comment by sghai** [comment]
I manually checked at server the output of hammer ping and got same issue, but after a few minutes it came up successfully.

```
~]# hammer -u admin -p changeme ping
candlepin:      
    Status:          ok
    Server Response: Duration: 27ms
candlepin_auth: 
    Status:          ok
    Server Response: Duration: 27ms
pulp:           
    Status:          ok
    Server Response: Duration: 27ms
foreman_tasks:  
    Status:          FAIL
    Server Response:

~]# hammer -u admin -p changeme ping
candlepin:      
    Status:          ok
    Server Response: Duration: 26ms
candlepin_auth: 
    Status:          ok
    Server Response: Duration: 27ms
pulp:           
    Status:          ok
    Server Response: Duration: 26ms
foreman_tasks:  
    Status:          ok
    Server Response: Duration: 1441ms
```

See the duration taken by `foreman_tasks`. This needs to be handled.

**Comment by sghai** [comment]
The ugly solution that I can think of putting a sleep here
https://github.com/RedHatSatellite/satellite-clone/blob/master/roles/sat6repro/tasks/main.yml#L97
` - katello-service start && sleep 3m`

**Comment by sthirugn** [comment]
ah yeah I had seen this problem.  Better to add some sleep here

**Comment by sthirugn** [comment]
Fixed in #48
